### ISSUE TITLE
1. No superfluous font-styles

### ISSUE THREAD

**Issue by fintelkai** [lx-unified-v2/lx-unified-v2#1]
> 1. Superfluous font-styles should be omitted. Do not use small caps for author/editor names, since they do not help to distinguish these from any other bits of information in the citation. In contrast, italics are worthwhile for distinguishing volume (book, journal, dissertation) titles [+ital] from article and chapter titles [-ital].

**Comment by fintelkai** [comment]
I think this can stand as is. I'd add that bold fonts are discouraged in general by most typographic experts. So, they should not be used either.

**Comment by haspelmath** [comment]
I don't think this is needed anymore. Some journals used to have smallcaps for names, but this is quite obsolete. In general, the 2007 version is a bit strange as it does not separate the rules from the justification of the rules. (I've heard people justify the use of smallcaps for surnames, to distinguish surnames and given names: this is very common in France and maybe other Mediterranean countries. So if one starts using justifications, one needs to counter other arguments etc., and hence I think one shouldn't do it, at least not as part of the rules.)

**Comment by Glottotopia** [comment]
I would be in favour of keeping this rule, just for the sake of clarity. Having a rule which everybody obeys anyway does no harm. If ever someone wants to use small caps in the future, we can point to this rule as established consensus and end the discussion there.
